### PR TITLE
list keys when select a db

### PR DESCRIPTION
**Pull request by jdneo**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/editors/rediscache/RedisExplorerEditor.java [hunk 1]
```diff
@@ -46,28 +47,41 @@
 import org.eclipse.swt.widgets.Text;
 import org.eclipse.swt.widgets.Button;
 import org.eclipse.swt.widgets.Table;
-import org.eclipse.swt.widgets.TableItem;
+import org.eclipse.swt.widgets.List;
```
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/editors/rediscache/RedisExplorerEditor.java]: List should be put before Table

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/rediscache/RedisExplorerPresenter.java [hunk 2]
```diff
@@ -55,7 +56,41 @@ public void onReadDbNum(String sid, String id) {
...
+     * @param id
+     *             resource id of Redis Cache
+     * @param db
+     *             index of Redis Cache database
+     * @param cursor
+     *             cursor for Redis Scan command
+     * @param pattern
+     *             pattern for Redis Scan Param
+     */
+    public void onDbSelect(String sid, String id, int db, String cursor, String pattern) {
```
Comment (adashen) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/rediscache/RedisExplorerPresenter.java]: do we need pattern and cursor here? 
cursor is String type?

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/editors/rediscache/RedisExplorerEditor.java [hunk 3]
```diff
@@ -172,6 +173,7 @@ public void renderDbCombo(int num) {
             cbDatabase.add(String.format(DBNameFormat, String.valueOf(i)));
         }
         cbDatabase.select(0);
+        redisExplorerPresenter.onDbSelect(sid, id, cbDatabase.getSelectionIndex());
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/editors/rediscache/RedisExplorerEditor.java]: also need to set text move to a separate method

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/rediscache/RedisExplorerPresenter.java [hunk 4]
```diff
@@ -71,9 +71,9 @@ public void onReadDbNum(String sid, String id) {
      * @param pattern
      *             pattern for Redis Scan Param
      */
-    public void onDbSelect(String sid, String id, int db, String cursor, String pattern) {
+    public void onDbSelect(String sid, String id, int db) {
         Observable.fromCallable(() -> {
-            return RedisExplorerMvpModel.getInstance().scanKeys(sid, id, db, cursor, pattern);
+            return RedisExplorerMvpModel.getInstance().scanKeys(sid, id, db, "", "");
```
Comment (adashen) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/rediscache/RedisExplorerPresenter.java]: scanKeys no need to convert empty string.

**Pull request merged**